app/src/main/java/com/arygm/quickfix/ui/Announcement.kt

```diff
@@ -26,56 +26,46 @@ import com.arygm.quickfix.ui.navigation.Route
 @Composable
 fun AnnouncementScreen(navigationActions: NavigationActions, isUser: Boolean, LoD: Boolean) {
 
-    val color1 = if (LoD) Color(0xFFF16138) else Color(0xFF633040)
-    val backgroundColor = if (LoD) Color.White else Color(0xFF282828)
+  val color1 = if (LoD) Color(0xFFF16138) else Color(0xFF633040)
+  val backgroundColor = if (LoD) Color.White else Color(0xFF282828)
 
-    // Use Scaffold for the layout structure
-    Scaffold(
-        containerColor = backgroundColor,
-        topBar = {
-            Surface(
-                shape = RoundedCornerShape(bottomStart = 16.dp, bottomEnd = 16.dp),
-                modifier = Modifier.fillMaxWidth()
-            ){
-                TopAppBar(
-                    title = {
-                        Box(modifier = Modifier.fillMaxWidth()) {
-                            Text(
-                                text = "Announcement",
-                                color = backgroundColor,
-                                modifier = Modifier.align(Alignment.Center)
-                            )
-                        }
-                    },
-                    colors = TopAppBarDefaults.topAppBarColors(
-                        containerColor = color1
-                    )
-                )
+  // Use Scaffold for the layout structure
+  Scaffold(
+      containerColor = backgroundColor,
+      topBar = {
+        Surface(
+            shape = RoundedCornerShape(bottomStart = 16.dp, bottomEnd = 16.dp),
+            modifier = Modifier.fillMaxWidth()) {
+              TopAppBar(
+                  title = {
+                    Box(modifier = Modifier.fillMaxWidth()) {
+                      Text(
+                          text = "Announcement",
+                          color = backgroundColor,
+                          modifier = Modifier.align(Alignment.Center))
+                    }
+                  },
+                  colors = TopAppBarDefaults.topAppBarColors(containerColor = color1))
             }
```

Comment:
The top app bar beeing present in a lot of screens can be abstracted in another file.